src/main/java/com/iyzipay/Options.java

```diff
@@ -5,6 +5,11 @@
     private String apiKey;
     private String secretKey;
     private String baseUrl;
+    private HttpClient httpClient;
```

Comment:
Options is a class for only api information. Putting HttpClient class into Options is not a good practice.